source/branding/index.html.md.erb

```diff
@@ -1,138 +1,150 @@
 ---
 title: GOV.UK Verify branding rules
-weight: 5
+weight: 6
 ---
 
 # GOV.UK Verify branding rules
+August 2018 
```

Comment:
@clairehansfordGDS I'm merging this in as is, but there's a feature in the Tech Docs Template that displays a banner at the bottom to say when the page was last reviewed: https://alphagov.github.io/tech-docs-manual/#last-reviewed-on-and-review-in

Happy to talk about how to use the feature 😄